src/main/java/com/exadel/discount/dto/tag/CreateTagDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.discount.dto.tag;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Data
+public class CreateTagDTO {
```

Comment:
I don't think you need separate DTO for this. If you want have separate rule - use validation groups